test-crates/viper/src/lib.rs

```diff
@@ -1,21 +1,73 @@
 duchess::java_package! {
+    package java.lang;
+    class Object {}
+
     package scala;
     class AnyVal { * }
     class Function1 {}
     class Tuple2<T1, T2> {}
 
+    package scala.collection;
+    interface IterableOnceOps<A, CC, C> {
+        default scala.collection.immutable.Seq<A> toSeq();
+    }
+    interface IterableOps<A, CC, C> extends scala.collection.IterableOnceOps<A, CC, C> {}
+    interface SeqOps<A, CC, C> extends scala.collection.IterableOps<A, CC, C> {}
+    interface StrictOptimizedSeqOps<A, CC, C> extends scala.collection.SeqOps<A, CC, C> {}
+
     package scala.collection.immutable;
     interface Seq<A> {}
 
+    package scala.collection.mutable;
+    class ArrayBuffer<A> implements scala.collection.StrictOptimizedSeqOps<
+        A, scala.collection.mutable.ArrayBuffer, scala.collection.mutable.ArrayBuffer<A>
```

Comment:
What is going on here? `ArrayBuffer` implements itself?

Reply:
To start, I guess `ArrayBuffer` is an interface? We should really fix that.

Reply:
...oh.

Reply:
Oh, and it includes HKT. Fun.

Reply:
If I replace the first `ArrayBuffer` on line 23 with the semantically equivalent `ArrayBuffer<java.lang.Object>`, then duchess fails with "declared interface `...` not found in the reflected interfaces (`...`)" because the check is syntactic. That's why I looked into how to generate a default argument.

Reply:
Hmm. It's not actually 100% semantically equivalent, although it doesn't matter here. That said, it seems *plausible* for us to extend the check?

Reply:
That could be a lot of work, I guess. Can you file an issue with some notes on this *particular* problem (it's a bit different than #57, I realize now) and the fact that you added defaults?